klipper-exporter.service

```diff
@@ -9,6 +9,7 @@ StartLimitIntervalSec=5
 [Service]
 User=pi
 WorkingDirectory=/home/pi/klipper-exporter
+EnvironmentFile=/home/pi/klipper-exporter/prometheus-klipper-exporter.env
```

Comment:
An alternative would be to just update the ExecStart line
```
ExecStart=/home/pi/klipper-exporter/prometheus-klipper-exporter -logging.level Warning
```

Reply:
I thought it could be useful to have this in an user-owned directory near the binary without the need to edit the SystemD unit directly.